local_planner/src/nodes/planner_functions.cpp

```diff
@@ -416,7 +420,7 @@ float costFunction(float e_angle, float z_angle, float obstacle_distance,
   // distance cost
   float distance_cost = 0.0f;
   if (obstacle_distance > 0.0f) {
-    distance_cost = 500.0f * 1.0f / obstacle_distance;
+    distance_cost = 1000.0f * 1.0f / obstacle_distance;
```

Comment:
The compiler might optimize this anyway, but why not just `1000.f / obstacle_distance` ?